app/static/css/themes/nord-light.css

```diff
@@ -0,0 +1,52 @@
+html {
+    background-color: #D8DEE9b3 !important;
+}
+
+body {
+    background-color: #f9fafb !important;
+}
+
+div {
+    /*background-color: #111 !important;*/
+    color: #3B4252 !important;
+}
+
+a:visited h3 div {
+    color: #A3BE8C !important;
+}
+
+a:link h3 div {
+    color: #4C566A !important;
+}
+
+a:link div {
+    color: #81a1c1 !important;
+}
+
+div span {
+    color: #3B4252 !important;
+}
+
+input {
+    background-color: #f9fafb !important;
+    color: #3B4252 !important;
+}
+
+#search-bar {
+    color: #4C566A !important;
+    background-color: #ffffff !important;
+}
+
+.search-container {
+    background-color: #f9fafb !important;
```

Comment:
Does this need to be specified? It should just inherit the color from the body...otherwise if it uses a different color it ends up looking like this:

![image](https://user-images.githubusercontent.com/33362396/95664298-6c59ad80-0b14-11eb-89fe-e47b701cb40d.png)